user_modules/shared_testcases/testCasesForPerformAudioPassThru.lua

```diff
@@ -0,0 +1,566 @@
+--This script contains common functions that are used in CRQs:
+-- [GENIVI] PerformAudioPassThru: SDL must support new "audioPassThruIcon" parameter
+--How to use:
+--1. local testCasesForPerformAudioPassThru = require('user_modules/shared_testcases/testCasesForPerformAudioPassThru')
+
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local applicationName = config.application1.registerAppInterfaceParams.appName
+local SDLConfig = require ('user_modules/shared_testcases/SmartDeviceLinkConfigurations')
+local PathToAppFolder = config.pathToSDL .. SDLConfig:GetValue("AppStorageFolder") .. "/" .. tostring(config.application1.registerAppInterfaceParams.appID .. "_" .. tostring(config.deviceMAC) .. "/")
+
+local testCasesForPerformAudioPassThru = {}
+
+function testCasesForPerformAudioPassThru:Check_audioPassThruIcon_Existence(self)
```

Comment:
@maria-mih, please provide description of function. As example:
--[[@Check_audioPassThruIcon_Existence: check that one of files "icon.png", ..... exist in storage folder
--! @parameters: NO
--]]